checkstyle/suppressions.xml

```diff
@@ -155,11 +149,7 @@
               files="StreamsPartitionAssignor.java"/>
 
     <suppress checks="NPathComplexity"
-              files="ProcessorStateManager.java"/>
-    <suppress checks="NPathComplexity"
-              files="StreamsPartitionAssignor.java"/>
-    <suppress checks="NPathComplexity"
-              files="StreamThread.java"/>
+              files="(ProcessorStateManager|InternalTopologyBuilder|StreamsPartitionAssignor|StreamThread).java"/>
```

Comment:
cc @mjsax @guozhangwang

Reply:
Also EosTestDriver below.

Reply:
@guozhangwang Please file a JIRA. :)

Reply:
Agreed. A TODO won't do and we will forget, too.